doc/cloud-init.md

```diff
@@ -126,6 +126,21 @@ Then send the following request:
       }
     }'
 
+### Configure `cloud-init` through the CLI
+
+If you are using the CLI to configure your instance, write the configuration to a file and include that in your request.
+For example, create `cloud-init.yml` with the following content:
+
+    #cloud-config
+    package_upgrade: true
+    packages:
+      - package1
+      - package2
+
+Then send the following request:
+
+    lxc config set <instance_name> cloud-init.user-data - < ~cloud-init.yml
```

Comment:
```suggestion
    lxc config set <instance_name> cloud-init.user-data - < cloud-init.yml
```

Sorry, not sure where that `~` came from in the input, it shouldn't be there. :)